lib/chef/provider/package/chocolatey.rb

```diff
@@ -211,11 +211,18 @@ def args_to_string(*args)
         def available_packages
           @available_packages ||=
             begin
+              name = 0
+              version = 1
```

Comment:
if i understand it, these are array indexes, and i think this is hindering readability more than its helping it...

its also plausible that these would wind up colliding with other variables/methods later and causing a bit of scope hell..